.npmignore

```diff
@@ -0,0 +1,7 @@
+*.tgz
+.github
+.travis.yml
+formatconfig.json
+gen-tsd.json
+test/
```

Comment:
Should this be `/test/`?

Reply:
I think the other ones should probably be prefixed with `/` too.

Reply:
Well, we've already started merging these into the other repos so I'm going to go ahead without changing this one in the `tedium-add-types` PRs at least.